Makefile

```diff
@@ -1,3 +1,3 @@
 gen:
-	easyjson notification.go response.go
+	easyjson notification.go api.go
```

Comment:
just moved code from `response.go` to `api.go` (and Reason to be typed) to make `go-apns` and `go-fcm` structures look similar